bip-0125.mediawiki

```diff
@@ -51,11 +51,11 @@ transaction) that spends one or more of the same inputs if,
 
 # The original transactions signal replaceability explicitly or through inheritance as described in the above Summary section.
 
-# The replacement transaction pays an absolute higher fee than the sum paid by the original transactions.
-
 # The replacement transaction does not contain any new unconfirmed inputs that did not previously appear in the mempool. (Unconfirmed inputs are inputs spending outputs from currently unconfirmed transactions.)
 
-# The replacement transaction must pay for its own bandwidth in addition to the amount paid by the original transactions at or above the rate set by the node's minimum relay fee setting.  For example, if the minimum relay fee is 1 satoshi/byte and the replacement transaction is 500 bytes total, then the replacement must pay a fee at least 500 satoshis higher than the sum of the originals.
+# The replacement transaction pays an absolute fee at least the sum paid by the original transactions.
```

Comment:
"an absolute fee at least" -> "an absolute fee **of** at least" (sounds better to my ears, but either way seems fine)